config/cmake/test_compiler_cxx/check_experimental_string_view.cpp

```diff
@@ -1,7 +1,7 @@
-#include <expirimental/string_view>
```

Comment:
Wow, it caught a bug with one of the checks.

Reply:
I know that surprised me too.  We had disabled experimental string view a long time ago since it has some compatibility issues, but it was still a bug.